src/easy-sql-test.js

```diff
@@ -39,6 +43,10 @@ export default class EasySQLTest {
   }
 
   _executeStorProc(storProcName, args = {}, callback) {
+    if (!storProcName) {
```

Comment:
we do this check for `_executeStorProc()` but not for `_query()` function. I think that we should do for both or for none.